contracts/core/EntryPoint.sol

```diff
@@ -227,7 +227,7 @@ contract EntryPoint is IEntryPoint, StakeManager, NonceManager, ReentrancyGuardT
             //address(1) is special marker of "signature error"
             require(
                 address(aggregator) != address(1),
-                "AA96 invalid aggregator"
+                FailedOp(totalOps, "AA96 invalid aggregator")
```

Comment:
Is `totalOps` the same as `opIndex` in this case? If it is not, any tool build around `FailedOp` will blame the wrong UserOp for this revert.